sound/soc/sof/topology.c

```diff
@@ -3263,7 +3263,7 @@ static int sof_route_load(struct snd_soc_component *scomp, int index,
 		sroute->private = connect;
 		sroute->src_widget = source_swidget;
 		sroute->sink_widget = sink_swidget;
-		sroute->setup = true;
+		sroute->setup = false;
```

Comment:
this has got nothing to do with multicore.

NAK NAK NAK.

Reply:
> this has got nothing to do with multicore.
> 
> NAK NAK NAK.

This is together with the change of making widgets/routes not loaded as default.